### PR TITLE
Cortx 32224 cicd fix

### PR DESCRIPTION
**Pull request by akshaym99**
# Problem Statement
- CICD: Stage wise duration and tests count

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by akshaym99** [comment]
@sonalk0209, please review.

File: scripts/cicd_k8s/run-sanity-multinode-rgw.groovy [hunk 1]
```diff
@@ -140,13 +140,14 @@ do
 			echo "te_id : $te_id"
 			echo "old_te : $old_te"
 			(set -x; python3 -u testrunner.py -te=$te_id -tp=$tp_id -tg=${Target_Node} -b=${Build_VER} -t=${Build_Branch} --force_serial_run ${Sequential_Execution} -d=${DB_Update} --xml_report True --validate_certs False)
+		    if [ -e "log/latest/failed_tests.log" ] ; then echo "Exists\n" >> stage_fail.log ; fi
```
[the remark this answers is not in the document]
Comment (akshaym99) [reply scripts/cicd_k8s/run-sanity-multinode-rgw.groovy:143]: For all other stages 'failed_tests.log' file is sufficient as there are only single TE. In regression multiple TEs.

**Review by sonalk0209 — APPROVED** [review]
Results look good.

**Comment by sonalk0209** [comment]
@akshaym99 please add job link where u had run this and the screenshot of final result.
Also add collections output.

**Comment by akshaym99** [comment]
Jenkins run: https://eos-jenkins.colo.seagate.com/job/Cortx-Deployment/job/Nightly-K8s-Deploy/847/ 
CICD

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**